### ISSUE TITLE
[Accessibility]A11y_VSCode_ScreenReaderOptimized_Keyboard: Focus is not clearly visible on Yes or No button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52589]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated: MAS2.4.7

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Navigate to File->select New File->Navigate to Status Bar->Click on "Screen Reader Optimized" button.
3. use tab or shift+tab key to move focus on Yes/ No button.

**Actual:**
Focus is not visible on Yes or No button. When focus moves on Yes or No button, slightly its color changes but unable to recognize where the focus is. Many of the times,color also changes on mouse hover.

**Expected:**
Focus should be outlined with a dotted line or solid line(majorly) so that it could be properly visible.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to view the focus properly.

**MAS Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

Attachment for Reference: 
![a11y vscode focus visible](https://user-images.githubusercontent.com/39204909/41714387-dbb8e630-756d-11e8-8195-27f21be126ce.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by bpasero** [comment]
Duplicate of https://github.com/Microsoft/vscode/issues/52676
